README.md

```diff
@@ -278,6 +278,7 @@ if (ripe.isOAuthPending()) {
 | `noMasks`         | *boolean*          | Used to negate the `useMasks` option                                                                                                                                                      |
 | `noPrice`         | *boolean*          | Used to negate the `usePrice` option                                                                                                                                                      |
 | `parts`           | *JSON Object*      | Defines the product initial parts. Each key is a part's name built with color and material information. Example: `var parts = { "sole": { "material": "nappa", "color": "white" }, ... }` |
+| `remoteOnConfig`  | *boolean*          | Activates the remote execution of the model's logic on config updates. True by default.                                                                                                                   |
```

Comment:
Document all of these

    this.remoteCalls = this.options.remoteCalls === undefined ? true : this.options.remoteCalls;
    this.remoteOnConfig =
        this.options.remoteOnConfig === undefined ? this.remoteCalls : this.options.remoteOnConfig;
    this.remoteOnPart =
        this.options.remoteOnPart === undefined ? this.remoteCalls : this.options.remoteOnPart;
    this.noDefaults = this.options.noDefaults === undefined ? false : this.options.noDefaults;